docs/sphinx/reference-scenes.rst

```diff
@@ -369,6 +369,18 @@ Scene Item Functions
 
 ---------------------
 
+.. function:: void obs_sceneitem_set_relative_pos(obs_sceneitem_t *item, const struct vec2 *pos)
+              void obs_sceneitem_set_relative_pos(const obs_sceneitem_t *item, struct vec2 *pos)
```

Comment:
Both of these function names say `set`. I imagine the second one should be get.

Reply:
Fixed.